k-distribution/include/kframework/kore

```diff
@@ -0,0 +1 @@
+../../../haskell-backend/src/main/native/haskell-backend/src/main/kore/
```

Comment:
What is this file used for? Searching for `kframework/kore` in this state-delta doesn't reveal anything where it's using this file for something AFAICT.

Reply:
This is a symlink to where the kore prelude is declared which is used in source builds.